src/libdrakvuf/drakvuf.c

```diff
@@ -133,10 +128,11 @@ void drakvuf_close(drakvuf_t drakvuf, const bool pause)
     GSList* loop = drakvuf->event_fd_info;
     while (loop)
     {
-        fd_info_t fd_info = (fd_info_t) loop->data;
+        fd_info_t fd_info = loop->data;
```

Comment:
Sorry, I meant there is no reason to even assign it to a local variable, you could just `g_free(loop->data)`

Reply:
You're right, and I should have caught that too.